### DISCUSSION TITLE
Using Redis for QueueDAO

### DISCUSSION THREAD

**Discussion by philipsamuel** [Netflix/conductor#3352] [Q&A]
Is it possible to use Redis for QueueDAO , Cassandra for persistence and Elastic for indexing. If we want to achieve this ,does it require customization or are there any configurations available

**Comment by v1r3n** [comment]
Hi @philipsamuel it does require a bit of customization, but it is possible.
Take a look at https://github.com/orkes-io/orkes-conductor-community which is the open source build of Conductor that uses redis for queues and postgres for workflow persistence.
